apps/bfd-shared-test-utils/src/main/java/gov/cms/bfd/DatabaseTestUtils.java

```diff
@@ -53,7 +53,7 @@ public final class DatabaseTestUtils {
   public static final String TEST_CONTAINER_DATABASE_IMAGE_PROPERTY = "its.testcontainer.db.image";
 
   /** The default test container image to use when nothing is provided. */
-  public static final String TEST_CONTAINER_DATABASE_IMAGE_DEFAULT = "postgres:14.9-alpine";
+  public static final String TEST_CONTAINER_DATABASE_IMAGE_DEFAULT = "postgres:16.2";
```

Comment:
Is there a reason we're not using alpine anymore?

Reply:
No, I suppose there's no reason not to use the alpine image. Fixed.